src/components/DatePicker/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bcrs-shared-components/date-picker",
-  "version": "1.1.9",
+  "version": "1.1.10",
```

Comment:
Revert this. Lerna will take care of it -- see [README](https://github.com/bcgov/bcrs-shared-components/blob/main/README.md) and make sure you follow the steps closely.

Reply:
Also, you need to rebuild and commit the StoryBook changes (again, see readme).

Reply:
ah okay sorry - thought that whole thing was for first time features

Reply:
Feel free to update the README to make it clearer :)

Reply:
alright I added a storybook update but it says npm run storybook:build `only required the first time per feature` -- I ran it and check it out, but there's nothing to commit. Do you still want me to run the storybook build? if so I'll update the readme too

Reply:
It won't hurt to run the SB build again.
You should have about a dozen changed files to commit.

```
npm run storybook:build
npm run storybook
```